static/app/components/events/autofix/autofixRootCause.tsx

```diff
@@ -345,6 +346,67 @@ function RootCauseContext({
   );
 }
 
+function formatRootCauseText(
+  cause: AutofixRootCauseData | undefined,
+  customRootCause?: string
+) {
+  if (!cause && !customRootCause) {
+    return '';
+  }
+
+  if (customRootCause) {
+    return customRootCause;
+  }
+
+  if (!cause) {
+    return '';
+  }
+
+  const parts: string[] = [];
+
+  // Add title
+  parts.push(cause.title);
+  parts.push('\n\n');
+
+  // Add description
+  parts.push(cause.description);
+
+  // Add code snippets if available
+  if (cause.code_context?.length > 0) {
+    parts.push('\n\nRelevant code:\n');
+    cause.code_context.forEach((context, index) => {
+      parts.push(`\nSnippet #${index + 1}: ${context.title}\n`);
+      if (context.description) {
+        parts.push(`${context.description}\n`);
+      }
+      if (context.snippet?.snippet) {
+        parts.push(`File: ${context.snippet.file_path}\n`);
+        parts.push('```\n');
+        parts.push(context.snippet.snippet);
+        parts.push('\n```\n');
+      }
+    });
+  }
+
+  return parts.join('');
+}
+
+function CopyRootCauseButton({
+  cause,
+  customRootCause,
+  isEditing,
+}: {
+  cause?: AutofixRootCauseData;
+  customRootCause?: string;
+  isEditing?: boolean;
+}) {
+  if (isEditing) {
+    return null;
+  }
+  const text = formatRootCauseText(cause, customRootCause);
+  return <CopyToClipboardButton size="sm" borderless text={text} />;
```

Comment:
looks a little weird to have a small borderless button next to larger bordered buttons, maybe make them all the same size?

Reply:
maybe make it a normal sized secondary bordered button?